tests/Tests.Json.Decode.fs

```diff
@@ -1003,9 +995,9 @@ Expecting an array but instead got: 1
                         """
 I run into the following problems:
 
-Error at: `$`
+Error at: `$[0]`
 Expecting a string but instead got: 1
-Error at: `$.test`
+Error at: `$[0]`
```

Comment:
It should be `$.[0]`

Reply:
Well, depending if you're thinking on the F# type or the JSON route. But I'm ok with changing it to `$.[0]` anyways :)